ministryofjustice/correspondence_tool_staff
# CT-2238 workflow states

## Description
Implement support for the available states for Offender SARs and provide an interface to move it through the workflow states. 

## Self-review checklist

 
### Screenshots
![state transitions](https://user-images.githubusercontent.com/1161161/61964036-e0b63880-afc4-11e9-84d2-ab55e10ee665.gif)

### Related JIRA tickets
https://dsdmoj.atlassian.net/browse/CT-2238
 
### Deployment
n/a
### Manual testing instructions
A newly created case should be able to be viewed and start off in "Data to be requested". You can then use the green button to move it through the states shown here. The current implementation throws an exception when you try to close the case - next ticket will get that working 

![image](https://user-images.githubusercontent.com/1161161/61964144-1c510280-afc5-11e9-8fe0-cf47428ab6e8.png)
